src/main/resources/org/jenkinsci/plugins/ewm/definitions/Template/help-label.html

```diff
@@ -1,3 +1,10 @@
 <div>
-    Node's label. Will use this template configuration for all the nodes matching the given label.
+    <p>
+        When the actual Node label (in the Node configuration) matches the <i>Node label</i> in this <i>External
```

Comment:
Actually, is it a single label? Does it support label expressions? E.g. "linux && 64bit"